packages-web/badge/src/components/Badge.tsx

```diff
@@ -1,31 +1,31 @@
-import { SFC, createElement } from "react";
+import { createElement, ReactElement } from "react";
 import classNames from "classnames";
 
 import "../ui/Badge.css";
 
 export interface BadgeProps {
-    badgeType: "badge" | "label";
+    type: "badge" | "label";
     defaultValue?: string;
     className?: string;
     style?: object;
     value?: string;
     bootstrapStyle?: BootstrapStyle;
     clickable?: boolean;
-    onClickAction?: () => void;
+    onClick?: () => void;
     getRef?: (node: HTMLElement) => void;
 }
 
 export type BootstrapStyle = "default" | "info" | "inverse" | "primary" | "danger" | "success" | "warning";
 
-export const Badge: SFC<BadgeProps> = props =>
+export const Badge = (props: BadgeProps): ReactElement =>
```

Comment:
convert to use tsx instead of createElement

Reply:
Done